tests/acceptance/bootstrap/OcisConfigContext.php

```diff
@@ -255,6 +317,10 @@ public function theAdministratorHasStartedServiceSeparatelyWithTheFollowingConfi
 	 * @throws GuzzleException
 	 */
 	public function rollback(): void {
+		if (\getenv('K8S') === "true") {
+			$this->rollbackK8sServices();
+			return;
+		}
```

Comment:
is this necessary? maybe we could just return
```suggestion
		if (\getenv('K8S') === "true") {
			return;
		}
```